src/OpenTelemetry.Exporter.InMemory/InMemoryExporterOptions.cs

```diff
@@ -0,0 +1,25 @@
+// <copyright file="InMemoryExporterOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+
+namespace OpenTelemetry.Exporter
+{
+    public class InMemoryExporterOptions
+    {
+        public IList<object> Trait;
```

Comment:
Public field! The horror! Should have `{ get; set; }` 😄 Might even want:
```csharp
public IList<object> Trait { get; set; } = new List<object>();
```
Not sure if you wanted it null by default?

Reply:
Oops, you're right, fixed.

Yes I put null/no-op by default.